vcpkg.cmake module errors in osx gcc

ITNOA

I got new error on travis with below message:

```
CMake Warning (dev) at third_party/vcpkg/scripts/buildsystems/vcpkg.cmake:182 (if):
  Policy CMP0057 is not set: Support new IN_LIST if() operator.  Run "cmake
  --help-policy CMP0057" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  IN_LIST will be interpreted as an operator when the policy is set to NEW.
  Since the policy is not set the OLD behavior will be used.
Call Stack (most recent call first):
  tests/CMakeLists.txt:13 (add_executable)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at third_party/vcpkg/scripts/buildsystems/vcpkg.cmake:182 (if):
  if given arguments:
    "MACOSX_BUNDLE" "IN_LIST" "ARGV"

  Unknown arguments specified
Call Stack (most recent call first):
  tests/CMakeLists.txt:13 (add_executable)

-- Configuring incomplete, errors occurred!

See also "/Users/travis/build/fnc12/sqlite_orm/compile/CMakeFiles/CMakeOutput.log".

The command "cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_TOOLCHAIN_FILE=../third_party/vcpkg/scripts/buildsystems/vcpkg.cmake
```

for more detailed please see https://travis-ci.org/fnc12/sqlite_orm/jobs/533558496

thanks

--- a/scripts/buildsystems/vcpkg.cmake
+++ b/scripts/buildsystems/vcpkg.cmake
@@ -169,6 +169,7 @@ function(add_executable name)
     _add_executable(${ARGV})
     list(FIND ARGV "IMPORTED" IMPORTED_IDX)
     list(FIND ARGV "ALIAS" ALIAS_IDX)
+    list(FIND ARGV "MACOSX_BUNDLE" MACOSX_BUNDLE_IDX)
     if(IMPORTED_IDX EQUAL -1 AND ALIAS_IDX EQUAL -1)
         if(VCPKG_APPLOCAL_DEPS)    
             if(_VCPKG_TARGET_TRIPLET_PLAT MATCHES "windows|uwp")
@@ -179,7 +180,7 @@ function(add_executable name)
                         -OutVariable out
                 )
             elseif(_VCPKG_TARGET_TRIPLET_PLAT MATCHES "osx")
-                if ("MACOSX_BUNDLE" IN_LIST ARGV)
+                if (NOT MACOSX_BUNDLE_IDX EQUAL -1)
                     add_custom_command(TARGET ${name} POST_BUILD
                     COMMAND python ${_VCPKG_TOOLCHAIN_DIR}/osx/applocal.py
                         $<TARGET_FILE:${name}>

